optimize.js

```diff
@@ -0,0 +1,79 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+async function viewDashboard([dashboardId]) {
+
+const accessToken = await getAccessToken();
+const optimizeApiUrl = process.env.OPTIMIZE_API_URL;
+const collectionId = process.env.OPTIMIZE_COLLECTION_ID;
+
+// This is the API endpoint to view your existing dashboard IDs
+const url = `${optimizeApiUrl}/public/dashboard?collectionId=${collectionId}`;
+
+    const options = {
+      method: "GET",
+      url,
+      headers: {
+        Accept: "application/json",
+        Authorization: `Bearer ${accessToken}`
+      }
+    };
+  
+    try {
+      const response = await axios(options);
+  
+      // Process the results from the API call.
+      const clientResponse = response.data;
+  
+      // Emit the dashboard details.
+      console.log("Dashboard:", clientResponse);
```

Comment:
Is there any data in the clientResponse that is valuable to parse out and emit, so that we aren't dumping JSON to the output?